pkg/utils/utils.go

```diff
@@ -21,8 +21,14 @@ import (
 	"log"
 	"os"
 	"path"
+	"syscall"
+	"unsafe"
+
+	"golang.org/x/sys/unix"
 )
 
+const getTermios = syscall.TCGETS
```

Comment:
Do you define this to make the code easier to understand? I'd argue that for people not acquainted with terminal details, `getTermios` is not easier than `TCGETS` so we could just use `syscall.TCGETS` in the code below.

In any case, feel free to ignore this nit.